<launch>
  <arg name="camera_name" default="camera" />
  <arg name="publish_cd" default="true" />
  <arg name="bias_file" default="" />
  <arg name="raw_file_to_read" default="" />
  <arg name="event_delta_t" default="0.000100" />

  <node pkg="prophesee_ros2_driver" exec="prophesee_ros2_driver" name="prophesee_ros2_driver" output="screen">
    <param name="camera_name" value="$(var camera_name)" />
    <param name="publish_cd" value="$(var publish_cd)" />
    <param name="bias_file" value="$(var bias_file)"/>
    <param name="raw_file_to_read" value="$(var raw_file_to_read)" />
    <param name="event_delta_t" value="$(var event_delta_t)" />
  </node>

</launch>
